build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -277,9 +277,19 @@ def glob(self, include, exclude=None, exclude_directories=1):
         raise NotImplementedError("Recursive globs not supported")
       # Bazel `*.mlir` glob -> CMake Variable `_GLOB_X_MLIR`
       var = "_GLOB_" + pattern.replace("*", "X").replace(".", "_").upper()
-      glob_vars.append(f"${{{var}}}")  # {{ / }} are the escapes for { / }
+      glob_vars.append(var)
       self.converter.body += f"file(GLOB {var} CONFIGURE_DEPENDS {pattern})\n"
-    return glob_vars
+    for pattern in exclude:
+      if "**" in pattern:
+        raise NotImplementedError("Recursive globs not supported")
+      exclude_var = ("_GLOB_" +
```

Comment:
We should really remove the CONFIGURE_DEPENDS - it adds nontrivial time to every single build. Near half of my incremental build time right now is spent globbing test directories and that kills me.

Really, we should stop doing what LLVM does and instead explicitly list test files in our BUILD (and thus cmake) files. Even cmake says not to use glob for files from the source tree: 
```
(We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.)
```

Reply:
(saw further on where you are now explicitly listing things, awesome!)

Reply:
Yup please respond to my comments on this issue in https://github.com/google/iree/issues/1083. Dropping the glob on test files is liable to mean tests don't get included. We can brainstorm various solutions.